src/main/java/easy/mind/com/api/сontroller/TestsQuestionsController.java

```diff
@@ -0,0 +1,43 @@
+package easy.mind.com.api.сontroller;
+
+import easy.mind.com.api.DTO.TestsQuestionsDTO;
+import easy.mind.com.api.entity.TestsQuestions;
+import easy.mind.com.api.service.TestsQuestionsService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users/{userId}/questions")
+@RequiredArgsConstructor
+public class TestsQuestionsController {
```

Comment:
let's split this controller into 2: one for questions (agnostic from user), second for user tests